### DISCUSSION TITLE
Email account sharing

### DISCUSSION THREAD

**Discussion by davide-alghi** [YetiForceCompany/YetiForce#64] [How to] [answered]
Hi all,
in YF7.0.2, I'm trying to understand how to share email account between users, but `Mail tools / Mail – Auto Login` panel is always empty.

**Comment by rskrzypczak** [comment, marked as answer]
Read this: https://github.com/YetiForceCompany/YetiForceCRM/discussions/17468#discussioncomment-9698174
In simple words:
- You cannot log in to private mailboxes, configured in user preferences, i.e. no one, except the user himself, has access to them.
- You can log in to public/corporate mailboxes, i.e. those added in the `Mail tools/Mail – Scanner` panel

**Comment by davide-alghi** [comment]
thanks Radosław,
now I'm able to set email account as public (in `Mail - Auto Login`), but I cannot find the way to switch between my private email account and public email accounts to use in RoundCube.

**Comment by rskrzypczak** [comment]
Check that you have the option to show a button in the header enabled
![image](https://github.com/user-attachments/assets/0a9816a7-4342-4c72-8b71-5bfbd7e44247)

and that roundcube is set as the default mail client in your user preferences.
![image](https://github.com/user-attachments/assets/b6799f9d-6cff-4110-bd33-2f22706da71a)
You should now have a button in the header that allows you to switch between mailboxes, just like it used to be.

**Comment by davide-alghi** [comment]
ok, found.
thanks so much Radosław, now I'm able.

BR
